eval/src/vespa/eval/instruction/dense_xw_product_function.cpp

```diff
@@ -1,23 +1,18 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
-#include "dense_xw_product_function.h"
-#include "dense_tensor_view.h"
+#include <vespa/eval/instruction/dense_xw_product_function.h>
```

Comment:
the header file should still be in the same directory as the cpp file and can be included with local syntax